razorpay-sdk/src/Utility.php

```diff
@@ -9,10 +9,20 @@ class Utility
     public function verifyPaymentSignature($attributes)
     {
         $expectedSignature = $attributes['razorpay_signature'];
-        $orderId = $attributes['razorpay_order_id'];
         $paymentId = $attributes['razorpay_payment_id'];
 
-        $payload = $orderId . '|' . $paymentId;
+        if (isset($attributes['razorpay_order_id']) === true)
+        {
+            $orderId = $attributes['razorpay_order_id'];
+
+            $payload = $orderId . '|' . $paymentId;
+        }
+        else
```

Comment:
@ccrims0n I considered that, but, if `razorpay_order_id` isn't set, then `razorpay_subscription_id` will surely be set, and I know that because I wrote the code for it. 

So IMO, it would be better the way it is now.